dev/make-distribution.sh

```diff
@@ -233,7 +233,7 @@ if [ "$MAKE_PIP" == "true" ]; then
   pushd "$SPARK_HOME/python" > /dev/null
   # Delete the egg info file if it exists, this can cache older setup files.
   rm -rf pyspark.egg-info || echo "No existing egg info file, skipping deletion"
-  python3 setup.py sdist
+  python setup.py sdist
```

Comment:
Can we keep python3 by default? I think the compatibility change was intentionally removed as of https://github.com/apache/spark/commit/04e99c1e1b29b691a8fb51ecfcd7e99482ee0bb3#diff-e77989ed21758e78331b20e477fc5582 to prepare the complete removal of Python 2 support.

Reply:
If so, we also do not need to add `from __future__ import print_function` in `python/setup.py` because it is always called by python3?

Reply:
Yeah, I think so.

Reply:
+1 for @HyukjinKwon 's advice.